cranelift/assembler-x64/meta/src/generate/inst.rs

```diff
@@ -194,7 +194,7 @@ impl dsl::Inst {
                 f.add_block(
                     "fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result",
                     |f| {
-                        for op in &self.format.operands {
+                        for op in self.format.operands.iter().filter(|o| !o.implicit) {
```

Comment:
One thing that can be a bit unfortunate here is that this implementation hides information of pre-register-allocation instructions that have implicit operands and the operands won't get printed. That seems like it could create a pretty surprising situation where an instruction uses an vreg operand but it doesn't show up in the disassembly.

Would it be possible to skip printing the operands post-regalloc but print them, maybe in a comment, pre-regalloc?

Reply:
Yup, made up some new syntax: `... ;; implicit: %rax`.